projects/base-library/project/.project

```diff
@@ -37,5 +37,10 @@
 			<type>2</type>
 			<locationURI>$%7BWORKSPACE_LOC%7D/com-module-firmware/projects/shared/test_framework</locationURI>
 		</link>
+		<link>
+			<name>test_framework</name>
+			<type>2</type>
+			<location>D:/PROJECTS/COMM_MODULE/projects/shared/test_framework</location>
```

Comment:
I see what the issue is. When you cloned the GitHub repository you named it "COMM_MODULE" instead of the default repo name "com-module-firmware". That's why the TestEngine link above line: `38` isn't working for you since you don't have a directory named `com-module-firmware`.


We can't merge to main what you have here since this is the absolute path specific to your machine. But if you rename "COMM_MODULE" to `com-module-firmware`, the TestEngine directory link above should work.


Notice that the TestEngine link uses the relative path from the workspace (which is/should be the`/PROJECT` directory in your machine - where the .metadata folder is).

This is a good finding actually, and it'll be nice to compile this information in the `tutorial\testing.md` file, under a new section labeled "Common Errors" or something.